packages/matter.js/package.json

```diff
@@ -24,45 +24,41 @@
     "url": "https://github.com/project-chip/matter.js.git"
   },
   "scripts": {
-    "clean": "rm -rf dist && rm -rf out && rm -f *.tsbuildinfo",
-    "build-cjs": "tsc -b tsconfig.dist-cjs.json && cp package-cjs.json dist/cjs/package.json",
-    "build-es": "tsc -b tsconfig.dist-es.json",
-    "build": "npm run build-cjs && npm run build-es",
-    "build-clean": "npm run clean && npm run build",
+    "clean": "build clean",
+    "build": "build",
+    "build-clean": "build --clean",
     "build-doc": "typedoc --excludeExternals --plugin typedoc-plugin-missing-exports --plugin typedoc-plugin-markdown --gitRemote upstream --tsconfig tsconfig.dist-cjs.json",
-    "test-node": "NODE_OPTIONS=--experimental-vm-modules jest",
-    "test-web": "karma start test/karma.conf.cjs",
-    "test": "npm run test-node && npm run test-web"
+    "test": "matter-test -w",
+    "coverage": "c8 matter-test esm"
   },
   "dependencies": {
     "bn.js": "^5.2.1",
     "elliptic": "^6.5.4"
   },
   "devDependencies": {
+    "@project-chip/matter.js-tools": "0.1.0",
```

Comment:
but tool ´s package is privte and will never be on npm ... so that might get issues if someone uses npm i after instalation or such other cases where devDeps are also installed?

Reply:
Not 100% sure but I believe it's OK because dev dependencies are only installed at build time.  The build would fail on the NPM package anyway because of missing files.

We could publish the tools package to be safe, that shouldn't hurt anything.

Reply:
we can leave out for now and add if needed ...